main.go

```diff
@@ -26,7 +26,7 @@ func main() {
 	if err != nil {
 		panic(fmt.Sprintf("error loading environment: %s", err))
 	}
-	if err := cf.SetEnvValues(env); err != nil {
+	if err := cf.SetCFOverrides(env); err != nil {
```

Comment:
Why bind implementation to CF?

Reply:
The "binding" is there from the time we removed the two main.go-s ( it's not a exactly a binding, but we override some properties if VCAP_APPLICATION is found)

Option two would be to have two main methods or additional flag to specify mode in which the service works